projects/core/src/cart/facade/multi-cart.service.ts

```diff
@@ -310,12 +314,30 @@ export class MultiCartService {
    * @param cartId
    * @param userId
    */
-  deleteCart(cartId: string, userId: string) {
+  deleteCart(cartId: string, userId: string): void {
     this.store.dispatch(
       new CartActions.DeleteCart({
         userId,
         cartId,
       })
     );
   }
+
+  /**
+   * Reloads the cart with specified id.
+   *
+   * @param cartId
+   * @param userId
+   */
+  reloadCart(cartId: string, extraData?: { active: boolean }): void {
```

Comment:
I really want to avoid using the other one as I don't want to introduce the added complexity of getting the `userId` to pass it to the function. It would mean I need to add the "user stream" and that makes everything more painful.

I think there is a place for both functions in the world. If you have the `userId` for instance in the active-cart service it makes sense to use the function where you pass it as parameter. So that way it's more options for people.